src/cat.sol

```diff
@@ -96,30 +99,30 @@ contract Cat is LibNote {
     // --- Administration ---
     function file(bytes32 what, address data) external note auth {
         if (what == "vow") vow = VowLike(data);
-        else revert();
+        else revert("Cat/file-wrong-param");
     }
     function file(bytes32 ilk, bytes32 what, uint data) external note auth {
         if (what == "chop") ilks[ilk].chop = data;
         else if (what == "lump") ilks[ilk].lump = data;
-        else revert();
+        else revert("Cat/file-wrong-param");
     }
     function file(bytes32 ilk, bytes32 what, address flip) external note auth {
         if (what == "flip") { ilks[ilk].flip = flip; vat.hope(flip); }
-        else revert();
+        else revert("Cat/file-wrong-param");
     }
 
     // --- CDP Liquidation ---
     function bite(bytes32 ilk, address urn) external returns (uint id) {
         (, uint rate, uint spot) = vat.ilks(ilk);
         (uint ink, uint art) = vat.urns(ilk, urn);
 
-        require(live == 1);
-        require(spot > 0 && mul(ink, spot) < mul(art, rate));
+        require(live == 1, "Cat/not-live");
+        require(spot > 0 && mul(ink, spot) < mul(art, rate), "Cat/not-unsafe");
```

Comment:
Wouldn't "not-safe" be more correct?

Reply:
The condition is that the Vault needs to be unsafe to be bitten.

Reply:
except it is checking that it is "unsafe" in order to bite it.  maybe `Cat/safe-no-bite` or `Cat/must-be-unsafe`?